package.json

```diff
@@ -13,18 +13,25 @@
   },
   "dependencies": {
     "bitsyntax": "~0.0.4",
+    "bluebird": "^3.4.6",
```

Comment:
Sorry for the shuffling here. Seems like `npm i --save` sorted entries alphabetically.